src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupLocationTests.cs

```diff
@@ -103,10 +103,13 @@ public void TestUpdateBackupLocation()
 
                 var backupLocation = client.BackupLocations.Get(ResourceGroupName, "local");
 
-                backupLocation.Path = @"\\100.68.73.194\Share";
-                backupLocation.UserName = @"Administrator";
+                backupLocation.Path = @"\\su1fileserver\SU1_Infrastructure_2\BackupStore";
+                backupLocation.UserName = @"azurestack\azurestackadmin";
                 backupLocation.Password = "password";
                 backupLocation.EncryptionKeyBase64 = "YVVOa0J3S2xTamhHZ1lyRU9wQ1pKQ0xWanhjaHlkaU5ZQnNDeHRPTGFQenJKdWZsRGtYT25oYmlaa1RMVWFKeQ==";
```

Comment:
You did randomly generate this key correct?

Reply:
You mean the encryption key? I didn't generate another random key but I think the original one in the test case should be randomly generated. Do you have any concern?

Reply:
Anyway, regenerated a random key and replaced the existing one. I don't find any other secrets exposed, let me know if you have any other concern.

Reply:
Looks good to me, just wanted to make sure before we push.